tox.ini

```diff
@@ -1,11 +1,14 @@
 [tox]
-envlist = py37,py38,py39,py310,mypy
+envlist = py37,py38,py39,py310,mypy,docs
 
 [testenv]
 deps =
     pytest
+    pytest-asyncio
     pytest-cov
-    pytest-mypy-testing==0.0.11
+    # https://github.com/davidfritzsche/pytest-mypy-testing/issues/35
+    git+https://github.com/davidfritzsche/pytest-mypy-testing.git@031514ff6ecd5bdf4d11ff238c14d4801b5e47f3
+    mypy==0.971
```

Comment:
Is mypy needed here now?

Reply:
I would get installed by the dependency above, but I think I added it here to freeze its version, in line with what we do in Scrapy.